integrations/email-notifier/integration.definition.ts

```diff
@@ -0,0 +1,26 @@
+import { IntegrationDefinition } from "@botpress/sdk";
+import { integrationName } from "./package.json";
+import { actions, events, states } from "src/definitions/index";
+
+export default new IntegrationDefinition({
+  name: integrationName,
+  version: "2.0.0",
+  title: "Email Notifier",
+  description: "Send emails to your Botpress bot in minutes",
```

Comment:
I think you mean from in this case?
```suggestion
  description: "Send emails from your Botpress bot in minutes",
```